testcontainers-dapr/src/test/java/io/dapr/testcontainers/DaprContainerTest.java

```diff
@@ -13,17 +13,17 @@ public void schedulerAndPlacementCustomImagesTest() {
 
     DaprContainer dapr = new DaprContainer(DAPR_RUNTIME_IMAGE_TAG)
             .withAppName("dapr-app")
```

Comment:
We could have used `DAPR_VERSION` instead of `1.15.7` to avoid repeating the version multiple times.